src/app/shared/models/workflow/workflow.model.ts

```diff
@@ -1,4 +1,18 @@
 export interface Workflow {
+    assignable: boolean;
+    commentable: boolean;
+    condition?: string;
+    icon: string;
     id: string;
     name: string;
+    nextAssign: string;
+    nextStep: string;
+    nextStepCurrentStep: boolean;
+    order: number;
+    owner?: string;
+    requiresCheckout: boolean;
+    roleHierarchyForAssign: boolean;
+    schemeId: string;
+    showOn: string[];
```

Comment:
This could be an enum

Reply:
We're not using it right now, but in the future it might be.